examples/mapbox-layer.js

```diff
@@ -1,83 +1,88 @@
 import Map from '../src/ol/Map.js';
 import View from '../src/ol/View.js';
 import Layer from '../src/ol/layer/Layer';
-import {assign} from '../src/ol/obj';
 import {toLonLat} from '../src/ol/proj';
-import SourceState from '../src/ol/source/State';
 import {Stroke, Style} from '../src/ol/style.js';
 import VectorLayer from '../src/ol/layer/Vector.js';
 import VectorSource from '../src/ol/source/Vector.js';
 import GeoJSON from '../src/ol/format/GeoJSON.js';
 
-class Mapbox extends Layer {
-
-  /**
-   * @param {import('../src/ol/layer/Layer').Options} options Layer options.
-   */
-  constructor(options) {
-    const baseOptions = assign({}, options);
-    super(baseOptions);
+const style = new Style({
+  stroke: new Stroke({
+    color: '#319FD3',
+    width: 2
+  })
+});
 
-    this.baseOptions = baseOptions;
+const vectorLayer = new VectorLayer({
+  source: new VectorSource({
+    url: 'data/geojson/countries.geojson',
+    format: new GeoJSON()
+  }),
+  style: style
+});
 
-    /**
-     * @private
-     * @type boolean
-     */
-    this.loaded = false;
+const map = new Map({
+  target: 'map',
+  view: new View({
+    center: [-10997148, 4569099],
+    zoom: 4,
+    minZoom: 1,
+    extent: [-Infinity, -20048966.10, Infinity, 20048966.10],
+    smoothExtentConstraint: false,
+    smoothResolutionConstraint: false
+  })
+});
 
-    this.initMap();
-  }
 
-  initMap() {
-    const map = this.map_;
-    const view = map.getView();
-    const center = toLonLat(view.getCenter(), view.getProjection());
+// init Mapbox object
 
-    const options = assign(this.baseOptions, {
-      attributionControl: false,
-      boxZoom: false,
-      center,
-      container: map.getTargetElement(),
-      doubleClickZoom: false,
-      dragPan: false,
-      dragRotate: false,
-      interactive: false,
-      keyboard: false,
-      pitchWithRotate: false,
-      scrollZoom: false,
-      touchZoomRotate: false,
-      zoom: view.getZoom() - 1
-    });
+const view = map.getView();
+const center = toLonLat(view.getCenter(), view.getProjection());
+const key = 'ER67WIiPdCQvhgsUjoWK';
 
-    this.mbmap = new mapboxgl.Map(options);
+const mbMap = new mapboxgl.Map({
+  style: 'https://maps.tilehosting.com/styles/bright/style.json?key=' + key,
+  attributionControl: false,
+  boxZoom: false,
+  center,
```

Comment:
minor: we should not use the es6 object key value shorthand
```suggestion
  center: center,
```